doc/iterations/2024.3/shapes/configurable_collapse.adoc

```diff
@@ -0,0 +1,47 @@
+= Configurable collapse
+
+== Problem
+
+Sirius Web doesn't provide a way for specifiers to set the default collapsible state of a node. It also doesn't provide a way to let specifiers define tools that collapse/expand nodes as part of other operations (e.g. to expand a collapsed node when a child node is created).
+
+Nodes are always created expanded, and the only way to collapse them right now is that the end user does it manually.
+
+== Key Result
+
+- A specifier must be able to define NodeDescriptions that are collapsed/expanded by default.
+- A specifier can define tools that collapse/expand other nodes.
+
+== Solution
+
+A new field _Default Collapsing State Expression_ should be displayed in NodeDescription. The field contains an expression which is evaluated on each diagram render to collapse/expand a node.
+
+This expression would be used to set the Collapsing State iff it hasn't been manually set by the end user or updated by a tool.
+
+A new operation _Set Collapsing State_ should be available in tools to expand/collapse nodes. This operation would have a _Graphical Candidate Expression_ field used to retrieve the nodes to expand/collapse, as well as a _CollapsingState_ field containing the value to set (Collapsed or Expanded).
```

Comment:
_Graphical Candidates Expression_ since it retrieves a lis of nodes?

Reply:
done